a_server.md

```diff
@@ -1,821 +1,707 @@
 
-Store the `logintoken`, `vendorname` and `vendor_pk` (the vendor's public key).  
-The user is now logged in and can move to the logged-in user menu items.
+The `sessionId` provided in the response should be used as the value of the
+`Session-Id` header in [authenticated requests](#authenticated-requests).
 
-##### N.B. The user's first access to their account data pulls that data from the B server. Until that happens their account data does not exist on the A server.
-##### Once they have made that initial access, the user has taken control of their data on the B server, and subsequent updates of that data will be pushed to the B server if the user's permissions allow it.
 
-#### Signing up an existing user with a new organization (aka vendor)
+<a name="login"></a>
 
-Obtain a signuptoken as above. On the registration form page, offer the user the alternative to add this organization to an existing account by logging into the existing account.
+### Login
 
-That login form should target /api/authn/register_add with the username, passphrase and signuptoken:  
-_POST https://sandbox.a.jlinclabs.net/api/authn/register_add_
+Endpoint:
+```
+POST /api/login
+```
 
+Post body:
 ```json
 {
- "username": "alice@example.com",
- "passphrase": "foobar",
- "signuptoken": "ec281d5348c24ed0d8637297ed9faad4"
+  "email": "example@example.com",
+  "password": "Pa$$w0rd",
+  "keepMeLoggedIn":true
 }
 ```
 
-On success the A server will return a login token and a `vendorPk` public key, which can be used to redirect the user to the new organization's page..
+Response body:
 
 ```json
 {
- "success": true,
- "logintoken": "6f81124583a583034df88a9664891a8b",
- "vendorPk": "JC0W885tID6b71XJ915uq-Ktqf9OFJgbu3iBNQ1IHnI"
+  "sessionId": "c97cb8cd5cc19177d4db61882ee258df",
+  "success": true
 }
 ```
 
+The `sessionId` provided in the response should be used as the value of the
+`Session-Id` header in [authenticated requests](#authenticated-requests).
+
+<a name="request-password-reset-email"></a>
 
-{% include starsep.html %}
+### Request Password Reset Email
 
-### Re-logging in
+Endpoint:
+```
+POST /api/reset_password/request
+```
 
-_POST https://sandbox.a.jlinclabs.net/api/authn/login_
+Post body:
 ```json
 {
- "username": "alice@example.com",
- "passphrase": "foobar"
+  "email": "example@example.com"
 }
 ```
-On success it returns
+
+Response body:
+
 ```json
 {
- "success": true,
- "token": "de7a126fef783bd6d9ac1da48cada93ee"
+  "success": true
 }
 ```
 
-{% include starsep.html %}
+<a name="reset-password"></a>
+
+### Reset Password
+
+Endpoint:
+```
+POST /api/reset_password
+```
 
-### Logging out
-_POST https://sandbox.a.jlinclabs.net/api/authn/logout_
+Post body:
 ```json
 {
- "token": "de7a126fef783bd6d9ac1da48cada93ee"
+  "resetPasswordToken": "GET_THIS_FROM_YOUR_EMAIL",
+  "password": "newPassword",
+  "passwordConfirmation": "newPassword"
 }
 ```
-On success it returns just:
+
```

Comment:
I'd add a sentence here that lets people know an email is sent.

Reply:
an email is sent here? Isnt this the enpoint that actually changes the password?